File: Java/FibonacciSeries.java
/** 
 * Recursive implementation of 
 * Fibonacci Series
*/

public class FibonacciSeries {
  
    // method to print the series 
    static int fib(int n) { 
        // Base Case 
        if (n <= 1) 
            return n; 
  
        // Recursive call 
        return fib(n - 1) 
            + fib(n - 2); 
    } 
  
    // main method 
    public static void main(String args[]){
        
        // N series 
        int N = 10; 
  
        // Print the first N numbers 
        for (int i = 0; i < N; i++) {
            System.out.print(fib(i) + " "); 
        }
        System.out.println();
    } 
} 

File: Go/FibonacciSeries.go
/** 
 * Recursive implementation of 
 * Fibonacci Series
*/
package main

import "fmt"

func fibonaci(i int) (ret int) {
	// Base Case
   if i == 0 {
      return 0
   }
   if i == 1 {
      return 1
   }
   // Recursive call
   return fibonaci(i-1) + fibonaci(i-2)
}

// main method
func main() {
	// N series
   var n int = 10
   var i int

   // Print the first N numbers
   for i = 0; i < n; i++ {
      fmt.Printf("%d ", fibonaci(i))
   }
}
